.github/workflows/rc-publish.yml

```diff
@@ -33,17 +33,15 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Set git config with NPM token
-        run: git config --global user.email "jack.howard@washpost.com" && git config --global user.name "jackhowa"
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_PERSONAL_AUTH_TOKEN }}
-
       - name: Get current datetime for re-running publish if necessary
         id: get-datetime
         run: echo "::set-output name=date::$(date +'%Y%m%d%H%M%S')"
 
       - name: Prerelease with tag, eg (`0.0.2-rc.{hash of commit}.{current-date-time}`)
-        run: npm version prerelease --preid=rc.$(git rev-parse --short HEAD).${{ steps.get-datetime.outputs.date }}
+        run: |
+          git config --local user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Comment:
Okay, I looked it up - but I would love for you to explain it to me :D

Github Actor: `The username of the user that initiated the workflow run.`

Does this mean whoever triggers the action will be the actor? And that would either manually from the github ui or triggering it through the local?

Reply:
yes! thanks for asking and discussing this in the meeting

Reply:
@RobinGiannattasio updated -- hope it's ok with the comment added to clarify your q.